show_char_code.py

```diff
@@ -11,4 +11,4 @@ def on_activated(self, view):
 
     def show_char_code(self, view):
         selected = view.substr(view.sel()[0].a)
-        view.set_status('charcode', "Char Code: {0}".format(ord(selected)))
```

Comment:
Hi, thanks for PR! I think it's better to use `{0:#x}` to display hex code. Also I noticed that in [fork of my repo](https://github.com/onelittlehope/ShowCharacterCode/blob/master/show_char_code.py) already added hex and byte codes. Maybe it's worth to do here the same so these changes would be available through Package Control. The only thing I'm not sure is the name "BYTE". Is that name correct for such character representation?